### PR TITLE
Tower_Netinstaller first check-in

### PR DESCRIPTION
**Pull request by justjais**
Tower Net_Installer first check-in and it includes:

- Added `install.yaml`
- Added roles having tower projects, inventory, user_accounts, teams, credentials as individual roles with respective tasks.
- Added tasks having inventory, users ans teams as task to create respective tower enteries.
- Also, added playbooks and changelog folder.

### CODE REVIEW

File: roles/tower_credentials/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (pabelanger) [inline roles/tower_credentials/.travis.yml:1]: We shouldn't be adding travis here,  we can great a job in zuul

File: roles/tower_inventory/.travis.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (pabelanger) [inline roles/tower_inventory/.travis.yml:1]: same comment

File: roles/tower_projects/.travis.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (pabelanger) [inline roles/tower_projects/.travis.yml:1]: same

File: roles/tower_team_accounts/.travis.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (pabelanger) [inline roles/tower_team_accounts/.travis.yml:1]: same

File: roles/tower_user_accounts/.travis.yml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (pabelanger) [inline roles/tower_user_accounts/.travis.yml:1]: same

**Review by pabelanger — CHANGES_REQUESTED** [review]

File: roles/tower_credentials/.travis.yml (hunk 1 above)
Comment (justjais) [reply roles/tower_credentials/.travis.yml:1]: sure, I'll remove the travis and update the PR

File: roles/tower_inventory/.travis.yml (hunk 2 above)
Comment (justjais) [reply roles/tower_inventory/.travis.yml:1]: same

**Pull request closed without merging**